OnlTools/Jevp/StJevpBuilders/fstBuilder.h

```diff
@@ -2,10 +2,7 @@
 #include <stdlib.h>
 
 #include "JevpBuilder.h"
-
-class daqReader;
-
-
+#include "DAQ_READER/daqReader.h"
```

Comment:
This is the reason why it can't compile in the CI environment. I suggest you keep the forward declaration for daqReader

Reply:
Thanks, I remembered the solution now....

Reply:
Just tried the forward declaration for daqReader on RCF but the compile failed.

Reply:
If the build on the farm failed we would see it [here](https://www.star.bnl.gov/public/comp/prod/Sanity/AutoBuild.html) or [here](https://www.star.bnl.gov/public/comp/prod/Sanity/AutoBuild-64bits.html)